### PR TITLE
upload Personal data from server

### PR DESCRIPTION
**Pull request by pinkevmladchy**

### CODE REVIEW

File: src/app/components/personal-cabinet/personal-cabinet.component.css [hunk 1]
```diff
@@ -31,4 +31,31 @@
...
+    background-color: lightblue;
+    height: 200px;
+    width: 200px;
+    text-align: center;
+    padding: 70px;
+}
+
+.names{
+    font-size: 40px;
 }
```
Comment (LizaShemetovska) [inline src/app/components/personal-cabinet/personal-cabinet.component.css]: Add new line

File: src/app/components/personal-cabinet/personal-cabinet.ts [hunk 2]
```diff
@@ -39,5 +46,14 @@ export class PersonalCabinetComponent implements OnInit {
...
+    this.accountService.getPersonalData()
+    .subscribe((data: any) => {
+      this.personalData = data;
+    });
+    this.accountService.getUserById()
+    .subscribe((data: any)=>{
+      this.userName = data.firstName[0]+data.lastName[0];
+    });
+  }
 }
```
Comment (LizaShemetovska) [inline src/app/components/personal-cabinet/personal-cabinet.ts]: Add new line

File: src/app/models/driverLicence.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { TransportCategories } from "./transportCategories";
+
+export class DriverLicence {
+    constructor(
+        public LicenseSerialNumber: string,
+        public IssuedBy: string,
+        public ExpirationDate: Date,
+        public UserCategories: Array<TransportCategories>
+    ) { }
+}
```
Comment (LizaShemetovska) [inline src/app/models/driverLicence.ts]: Add new line

**Review by LizaShemetovska — CHANGES_REQUESTED** [review]

File: src/app/components/personal-cabinet/personal-cabinet.component.html [hunk 4]
```diff
@@ -7,13 +7,20 @@
...
+        <div *ngIf="this.isActivInfo">
+          <div nz-row class="persData">
+            <div nz-col nzSpan="8">
+              <div class="circle"><span class="names">{{this.userName}}</span></div>
+            </div>
+            <div nz-col nzSpan="16">
+              <h1><span>IPN:</span>{{this.personalData.ipn}}</h1>
+              <h1><span>Adress:</span>{{this.personalData.address}}</h1>
+              <h1><span>Service number:</span>{{this.personalData.serviceNumber}}</h1>
+              <h1><span>Birthday:</span>{{this.personalData.birthDay}}</h1>
```
Comment (LizaShemetovska) [inline src/app/components/personal-cabinet/personal-cabinet.component.html:19]: Need to use elements from Ant design

File: src/app/models/adress.ts [hunk 5]
```diff
@@ -0,0 +1,12 @@
+export class Address {
+    constructor(
+        public Country:string,
+        public Region:string,
+        public City:string,
+        public District :string,
+        public Street:string ,
+        public Building:string,
+        public Appartament:number,
+        public PostalCode:string
```
Comment (NerevitVasyan) [inline src/app/models/adress.ts]: add space after :

File: src/app/components/personal-cabinet/personal-cabinet.ts [hunk 6]
```diff
@@ -12,8 +14,13 @@ export class PersonalCabinetComponent implements OnInit {
   isActivetead = false;
   isActivTransport = false;
   isActivViolation = false;
-  
-  constructor(private router: Router) { }
+  public personalData: any;
```
Comment (NerevitVasyan) [inline src/app/components/personal-cabinet/personal-cabinet.ts]: please, try to use models for this

File: src/app/components/personal-cabinet/personal-cabinet.ts [hunk 7]
```diff
@@ -12,8 +14,13 @@ export class PersonalCabinetComponent implements OnInit {
...
+  public user: any;
+  public userName: null;
```
Comment (NerevitVasyan) [inline src/app/components/personal-cabinet/personal-cabinet.ts]: specify type, i suppose here should be string

**Review by vladtymo — APPROVED** [review]

**Review by NerevitVasyan — APPROVED** [review]

**Review by LizaShemetovska — APPROVED** [review]

**Pull request merged**